ale/util.py

```diff
@@ -132,31 +132,54 @@ def read_pref(path):
                 pvlprefs = pvl.loads(preftext)
         except Exception as e:
             raise Exception(f'Failed to load IsisPreferences file [{path}]: {e}')
-            
+
         return pvlprefs
 
     argprefs = {}
     if isis_preferences:
         if isinstance(isis_preferences, dict):
             argprefs = isis_preferences
         elif os.path.isfile(isis_preferences):
-            argprefs = reaf_pref(isis_preferences)
+            argprefs = read_pref(isis_preferences)
         else:
             warnings.warn(f'{isis_preferences} does not exist, ignoring')
 
     try:
         homeprefs = read_pref(os.path.join(os.path.expanduser("~"), '.Isis', 'IsisPreferences'))
-    except: 
+    except Exception as e:
+        print(f'error: {e}')
         homeprefs = {}
 
-
-    isisrootprefs = read_pref(os.path.join(os.environ["ISISROOT"], 'IsisPreferences'))
+    try:
+        isisroot = os.environ['ISISROOT']
+        print(isisroot)
+        isisrootprefs = read_pref(os.path.join(os.environ["ISISROOT"], 'IsisPreferences'))
+    except Exception as e:
+        print('failed {e}')
```

Comment:
Same here, saying the error happened while trying to read the ISISROOT preferences would be helpful.

Reply:
same as above.